src/services/status.service.test.js

```diff
@@ -58,29 +62,129 @@ describe("status.service getStatus()", () => {
   });
 });
 
-describe("status.service setStatus()", () => {
+describe("Function: status.service setStatus()", () => {
   let result;
 
   describe("given a the supplied statusName does not exist", () => {
     beforeEach(async () => {
+      jest.clearAllMocks();
+      getStoredStatus.mockImplementation(() => ({}));
       updateStoredStatus.mockImplementation(() => "new value");
+      getEmailDistribution.mockImplementation(() => ['test@test.com']);
       result = await setStatus("newStatusItem", "new value");
     });
 
     it("should return the new value of the status name", () => {
       expect(result).toBe("new value");
     });
+
+    it("should call mocks with the correct values to send an email", () => {
+      expect(getEmailDistribution).toHaveBeenCalledTimes(1);
+      expect(sendSingleEmail).toHaveBeenCalledTimes(1);
+      expect(sendSingleEmail).toHaveBeenLastCalledWith(
+        "e6692529-52a3-45cb-96b7-0c8cfe282167",
```

Comment:
If this value is not set in .env, this and two other tests fail. Greater isolation is required in unit tests.

Reply:
Fixed unit tests to not rely on this value